cfgmgr/intfmgr.cpp

```diff
@@ -616,6 +618,31 @@ bool IntfMgr::isIntfStateOk(const string &alias)
     return false;
 }
 
+void IntfMgr::delIpv6LinkLocalNeigh(const string &alias)
+{
+    vector<string> neighEntries;
+
+    SWSS_LOG_INFO("Deleting ipv6 link local neighbors for %s", alias.c_str());
+
+    m_neighTable.getKeys(neighEntries);
+    for (auto neighKey : neighEntries)
+    {
+        if (!neighKey.compare(0, alias.size(), alias.c_str()))
+        {
+            vector<string> keys = tokenize(neighKey, ':', 1);
+            if (keys.size() == 2)
+            {
+                IpAddress ipAddress(keys[1]);
+                if (ipAddress.getAddrScope() == IpAddress::AddrScope::LINK_SCOPE)
+                {
+                    m_neighTableProducer.del(neighKey);
```

Comment:
This is not a good approach as to have multiple producers for the same table APP_NEIGH_TABLE_NAME (Owned by neighsyncd). I think, intfmgr should delete this from kernel directly. Is that feasible?

Reply:
@prsunny I do agree about multiple producers to the same table.
If intfmgr deletes the kernel entry, then neighsync have to delete that entry from APPL_DB even though the link-local mode is disabled right. Added those changes, please continue the review.